Library/test-helpers/lib.sh

```diff
@@ -2032,6 +2284,9 @@ limeLibraryLoaded() {
         rlLogError "Failed loading library keylime/test-helpers."
         return 1
     fi
+
+    echo -e "\nInstalled requiered packages for helping library"
+    dnf install -y nmap
```

Comment:
can you add also other packages used in a library (tpm2-tools, openssl, beakerlib, maybe podman???) and install them only when they are not installed?
`rpm -q $PACKAGES || yum -y install $PACKAGES`